friendsofphp/php-cs-fixer/2.2/.php_cs.dist

```diff
@@ -1,8 +1,14 @@
 <?php
 
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
```

Comment:
`->(__DIR__.'/src')`

Reply:
Haha, seems like we posted at the same time, I also suggested `tests/` :)